### DISCUSSION TITLE
Is it possible to map two paths to two ports in a single docker container?

### DISCUSSION THREAD

**Discussion by avowkind** [umputun/reproxy#76] [Q&A]
I have a docker image that has an API port and Admin port.  8080, 8081.  I want to map these to /api  and /apiadmin.  
Is this possible using the docker labels approach?

**Comment by umputun** [comment]
each container supports a single port and a single source (path) with labels, so the answer is no

**Reply by umputun** [reply]
update: it is possible to do now, with multiple sets of reproxy.N.something labels introduced  [here](https://github.com/umputun/reproxy/issues/78)

**Comment by umputun** [comment]
however it is possible to do with the static of file provider
